charts/redpanda/values.yaml

```
@@ -640,25 +662,16 @@ listeners:
         # -- If undefined, `listeners.kafka.external.default.port` is used.
         advertisedPorts:
         - 31092
-  # -- HTTP API listeners (aka PandaProxy).
-  http:
-    enabled: true
-    port: 8082
-    kafkaEndpoint: default
-    tls:
-      # enabled: true
-      cert: default
-      requireClientAuth: false
-    external:
-      default:
-        # enabled: true
-        port: 8083
-        advertisedPorts:
-        - 30082
+        # -- Uncomment to define external tls
+        # tls: #
+        #   Optional flag to override the global TLS enabled flag.
+        #   enabled: true
+        #   cert: external
```

```suggestion
        # tls:
        #   # Optional flag to override the global TLS enabled flag.
        #   # enabled: true
        #   cert: external
```

This allows most editors to easily uncomment the block and still have the comment and optional parameter commented.